Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,41 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+/*
+    Abstract class to be inherited by a local player, networked player, client player?, and AI player
+ */
+
+public /* abstract */ class Player : MonoBehaviour
```

Comment:
is there a reason abstract is still commented?

Reply:
ye cause in the gamemanager right now it just has two player classes added at run time for super basic testing. if you run the main scene you'll see two players attached and whichever's turn it has an editor button to finish their turn

Reply:
got it, so keeping it not abstract until we make the other player classes so that the demo you have now works